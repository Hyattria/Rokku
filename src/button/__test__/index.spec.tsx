import React from 'react';
import { mount } from 'enzyme';
import Button from '../index';

import mountTest from '../../../tests/shared/mountTest';

describe('Button', () => {
  mountTest(Button);
  mountTest(() => <Button type="danger" />);

  it('renders correctly', () => {
    expect(<Button />).toMatchRenderedSnapshot();
  });

  it('mount correctly', () => {
    expect(() => mount(<Button />)).not.toThrow();
  });
});
