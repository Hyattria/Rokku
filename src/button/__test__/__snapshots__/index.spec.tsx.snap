// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  class="rk-button rk-button--default rk-button--normal"
/>
`;
